salesagility/SuiteCRM
# Fixed #7298 - Emails 'Bulk Action' is disabled after upgrade to 7.10.16

## Description



Fixes the email list-view bulk action from not appearing when selecting records. Also fixes the list-view showing 21 items by default instead of 20.

## Motivation and Context

Issue reference: #7298

## How To Test This

1. Setup inbound email.
2. Go to the email-listview and try to use the bulk action menu.
3. Check that selecting multiple emails still works correctly for bulk action options.

## Types of changes


### Final checklist
